rules/library.bzl

```diff
@@ -354,9 +354,9 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
     cpp_sources = []
     public_headers = kwargs.pop("public_headers", [])
     private_headers = kwargs.pop("private_headers", [])
-    objc_hdrs = [f for f in public_headers if f.endswith((".h", ".hh"))]
+    objc_hdrs = [f for f in public_headers if f.endswith((".h", ".hh", ".hpp"))]
```

Comment:
Thanks for the PR! Perhaps move this to a var - there's also a few other "header like" files out there we may want to add e.g. https://github.com/pinterest/PodToBUILD/blob/db17580e4d6c4f852163357310af588cd29531c0/Sources/PodToBUILD/RuleUtils.swift#L54